app/src/main/java/com/nervousfish/nervousfish/modules/qr/QRGenerator.java

```diff
@@ -64,42 +63,33 @@ public static Bitmap encode(final String publicKey) {
             LOGGER.error("Failed to encode bitmap", e);
         }
 
-        return bitmap;
-    }
+        final Matrix matrix = new Matrix();
 
-    /**
-     * Decodes the QRcode to a public key string.
-     *
-     * @param qrCode The QR code as a bitmap
-     * @return The decoded public key.
-     * @throws IOException If the QR code could not be decoded.
-     */
-    // We suppress this warning because an EnumMap is much more efficient than a hashmap for such small maps
-    @SuppressWarnings("PMD.UseConcurrentHashMap")
-    public static String decode(final Bitmap qrCode) throws IOException {
-        final Reader qrReader = new MultiFormatReader();
-        final int[] intArray = new int[qrCode.getWidth() * qrCode.getHeight()];
+        // resize the bit map
+        matrix.postScale(RESIZE_QR_CODE, RESIZE_QR_CODE);
+        final Bitmap largerCode = Bitmap.createBitmap(bitmap, 0, 0, bitmap.getWidth(), bitmap.getHeight(), matrix, true);
```

Comment:
OK, fair point 😅

Anyways, the variable is kinda redundant since you're immediately returning it afterwards... Can you remove the variable declaration and just return the value? 😇